translate/convert/po2l20n.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2016 Zuza Software Foundation
+#
+# This file is part of translate.
+#
+# translate is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# translate is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+
+"""Convert Gettext PO localization files to .ftl files.
+"""
+
+import logging
+
+from translate.convert import convert
+from translate.storage import po, l20n
+
+
+logger = logging.getLogger(__name__)
+
+
+class po2l20n(object):
+
+    def __init__(self, inputfile, outputfile, templatefile=None,
+                 includefuzzy=False, outputthreshold=None):
+
+        if templatefile is None:
+            raise ValueError("must have template file for ftl files")
```

Comment:
Why? The PO file has both the source and the target, so it is perfectly possible to just get a l20n file from it without the need of any l20n template. Using a l20n template is only necessary when you want the final l20n to align with l20n templates.